src/Pages/Login.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+export default function Login() {
+
+	const history = useHistory()
+	
+	function login() {
+		localStorage.setItem("isAuthenticated", true)
+		history.replace("/dashboard")
+	}
+
+
+	return (
+		<>
+			<br />
+			<br /> <br /> <br />
+			<h1>Login Page</h1>
+			<br />
+			<label> user name </label>
+			<input type="text"  />
+			<br />
+			<br />
+			<label> password </label>
+			<input type="text" />
+			<br />
```

Comment:
The pw should not be visible from the oustide but hided with dots or something. 
input type="password"
![Screen Shot 2020-11-05 at 12 38 43](https://user-images.githubusercontent.com/58600732/98236580-e2123700-1f63-11eb-8dc8-92efeb18c482.png)

Reply:
ah yeah, I just created the easiest thing I could, just so the router works. Now we can start working on this